src/haosTearDownOfDesignPatterns/AdaptorPattern.java

```diff
@@ -0,0 +1,50 @@
+package haosTearDownOfDesignPatterns;
```

Comment:
Throw this package in the `extras` folder at the root of the project, this shouldn't be in `src`